lib/crypto.nit

```diff
@@ -183,6 +183,36 @@ redef class Text
 	end
 end
 
+redef class Bytes
+
+	# Returns `self` xored with `key`
+	#
+	# The shortest of the two is cycled through until the longest has been
+	# completely xored.
```

Comment:
also rename the method `xorcipher` or else people will mistake it with the mathematical bitwise xor operation (ie without cycling).